sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -84,6 +86,17 @@ def get_magic_prompt_device() -> torch.device:
     return device
 
 
+def requote_prompt(prompt: str) -> str | None:
+    # if prompt is empty, return None
+    # if prompt will not be auto quoted by webui and can effected by stripping, we need to quote it ourselves
+    if prompt:
+        if prompt != prompt.strip() and prompt == generation_parameters_copypaste.quote(
+            prompt,
+        ):
+            prompt = json.dumps(prompt, ensure_ascii=False)
```

Comment:
Shouldn't this just use the result from `quote(prompt)` instead of duplicating that logic here?